internal/key/key.go

```diff
@@ -100,6 +100,11 @@ func IsPureCAPIProvider(provider string) bool {
 	return contains(PureCAPIProviders(), provider)
 }
 
+// IsCAPIProviderThatUsesReleases returns whether a given provider is a CAPI provider that uses new releases.
+func IsCAPIProviderThatUsesReleases(provider string) bool {
```

Comment:
what about renaming this function to `IsProviderUsingReleases`

Reply:
it's a bit easier to read IMHO

Reply:
or IsCAPIProviderUsingReleases

Reply:
✅ renamed it